.gitlab-ci.yml

```diff
@@ -194,7 +194,7 @@ test_cpp_mac_10_13:
     - export VIRTUALENV="/Library/Frameworks/Python.framework/Versions/2.7/bin/virtualenv"
     - (test -d debug && test -d release) || ./configure --no-python
     - cd debug/test
-    - make -j8
+    - bash -c "make -j8 2>&1 | grep -v -E 'ld[^a-z]*warning[^a-z]*direct access'"
```

Comment:
Could we either start the expression with a `^` or end it with a `$`?

Same question for line 216.